404.html

```diff
@@ -5,6 +5,95 @@
   <script type="text/javascript">
 	window.isErrorPage = true;
 	window.errorCode = '404';
+
+	const redirectMap = new Map(
+		[
+			['ae_ar', '/'],
```

Comment:
Suggestion: remove all entries that map to `/`, and make `/` the default value if the entry is not found in `redirectMap`. That will save a few bytes.

Reply:
@ramboz, @meryllblanchet We need to set the `international` cookie also, only for these valid locales. For example, if the user is trying to go to www.adobe.com/in/express/pricing, we need to set the `international` cookie to `in` and then load www.adobe.com/express/pricing. If we remove `/in/, /` altogether, then we don't have the value for cookie. We can simply set the locale obtained from the url as cookie, but we'll end up setting invalid cookies like `express` (when the user hits something like www.adobe.com/express/abc). So, I'd like to keep these mappings as-is.

WDYT?

cc: @sirgil2009